react-quiz/src/App.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react';
+import { fetchQuizQuestions } from './API';
+// Components
+import QuestionCard from './components/QuestionCard';
+// types
+import { QuestionsState, Difficulty } from './API';
+// Styles
+import { GlobalStyle, Wrapper } from './App.styles';
+
+export type AnswerObject = {
+  question: string;
+  answer: string;
+  correct: boolean;
+  correctAnswer: string;
+};
+
+const TOTAL_QUESTIONS = 10;
+
+const App: React.FC = () => {
+  const [loading, setLoading] = useState(false);
+  const [questions, setQuestions] = useState<QuestionsState[]>([]);
+  const [number, setNumber] = useState(0);
+  const [userAnswers, setUserAnswers] = useState<AnswerObject[]>([]);
+  const [score, setScore] = useState(0);
+  const [gameOver, setGameOver] = useState(true);
+
+  const startTrivia = async () => {
+    setLoading(true);
+    setGameOver(false);
+    const newQuestions = await fetchQuizQuestions(
+      TOTAL_QUESTIONS,
+      Difficulty.EASY
+    );
+    setQuestions(newQuestions);
+    setScore(0);
+    setUserAnswers([]);
+    setNumber(0);
+    setLoading(false);
+  };
+
+  const checkAnswer = (e: any) => {
```

Comment:
dont use `any` unless you actually dont know the type.

Also add return type on function - void